pkg/controller/storagecluster/reconcile.go

```diff
@@ -1152,3 +1156,128 @@ func getMonCount(nodeCount int) int {
 
 	return count
 }
+
+// ensureJobTemplates ensures if the osd removal job template exists
+func (r *ReconcileStorageCluster) ensureJobTemplates(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	osdCleanUpTemplate := &openshiftv1.Template{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      sc.Name + "-osd-removal",
```

Comment:
Will the template be called `openshift-storage-osd-removal`? Or what is `sc.Name`? How about if we just use a constant for the template name? Then the documentation can use the well-known template name.

Reply:
This makes sense, it would simplify the documentation. I propose something simple like `ocs-osd-removal`.